packages/javascript/bh-shared-ui/src/components/Disable2FADialog/Disable2FADialog.tsx

```diff
@@ -15,19 +15,28 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@mui/material';
-import React, { useState } from 'react';
+import React from 'react';
 
 const Disable2FADialog: React.FC<{
     open: boolean;
     onClose: () => void;
     onCancel: () => void;
     onSave: (secret: string) => void;
     error?: string;
-}> = ({ open, onClose, onCancel, onSave, error }) => {
-    const [secret, setSecret] = useState('');
+    secret: string;
+    setSecret: (secret: string) => void;
+    contentText: string;
+}> = ({ open, onClose, onCancel, onSave, error, secret, setSecret, contentText }) => {
+
 
     const handleOnClose = () => {
         onClose();
+        setSecret('');
+    };
+
+    const handleOnCancel = () => {
+        onCancel();
+        setSecret('');
```

Comment:
This may be leftover from lifting the state out of this component. It would be more consistent to handle clearing the secret in the `onClose` and `onCancel` handlers passed into this component in `Users.tsx` like it was done for `onSave`.